File: Back/src/ProEvents.Domain/Lecturer.cs
using System.Collections.Generic;

namespace ProEvents.Domain
{
    public class Lecturer
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Resume { get; set; }
        public string IamgeUrl { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public IEnumerable<SocialNetwork> SocialNetworks { get; set; }
        public IEnumerable<LecturerEvent> LecturerEvents { get; set; }



    }
}

File: Front/ProEvents-App/src/app/models/Lecturer.ts
import { Event } from "./Event";
import { SocialNetwork } from "./SocialNetwork";

export interface Lecturer {
  id:number;
  name:string;
  resume:string;
  iamgeUrl:string;
  phone:string;
  email:string;
  socialNetworks:SocialNetwork[];
  lecturerEvents:Event[];


}
